pkg/v1/sdk/features/plugin/activate.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package plugin
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// FeatureActivateCmd is for activating Features
+var FeatureActivateCmd = &cobra.Command{
+	Use:   "activate <feature>",
+	Short: "Activate Features",
+	Args:  cobra.ExactArgs(1),
+	Example: `
+	# Activate a cluster Feature
+	tanzu feature activate myfeature`,
+	RunE: featureActivate,
+}
+
+func init() {
+	FeatureActivateCmd.Flags().StringVarP(&featuregate, "featuregate", "f", "tkg-system", "Activate a Feature gated by a particular FeatureGate")
+}
+
+func featureActivate(cmd *cobra.Command, args []string) error {
```

Comment:
Related to previous comment about package organization: IMO it's better to keep these functions in `cmd/cli/plugin/feature` and only have generic `get/set/list feature` functions in `pkg/v1/sdk/features`. I don't think this package needs to be aware of CLI-related things.